btk/measure.py

```diff
@@ -135,18 +137,107 @@ def basic_measure(
     return {"catalog": catalog}
 
 
-def sep_measure(
+def sep_multiband_measure(
     batch,
     idx,
     channels_last=False,
     surveys=None,
+    matching_threshold=1.0,
     sigma_noise=1.5,
     is_multiresolution=False,
     **kwargs,
 ):
-    """Return detection, segmentation and deblending information with SEP.
+    """Returns centers detected with source extractor by combining predictions in different bands.
```

Comment:
Could you add an explanation to the docs explaning exactly how you 'combine' the predictions in different bands?

Reply:
Sure! Added in the new commit.

Reply:
Thanks!